parsers/jscript.c

```diff
@@ -1940,7 +1950,6 @@ static bool parseStatement (tokenInfo *const token, bool is_inside_class)
 	bool is_const = false;
 	bool is_terminated = true;
 	bool is_global = false;
-	bool has_methods = false;
```

Comment:
To extract an object having a method or a property as an anonymous tag, we must keep this local variable. See below.